master/getting-started/kubernetes/installation/vagrant/Vagrantfile

```diff
@@ -12,7 +12,7 @@ Vagrant.configure("2") do |config|
   config.ssh.insert_key = false
 
   config.vm.box = "coreos-%s" % update_channel
-  config.vm.box_version = ">= 1122.0.0"
+  config.vm.box_version = ">= 1576.4.0"
```

Comment:
Looks like 1576.4.0 is the "latest" release.

Reply:
yeah, in the stable channel